<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyLuxCore Tool NetConsole</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTabWidget" name="tabWidgetMain">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="currentJob">
        <attribute name="title">
         <string>Current job</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QScrollArea" name="scrollArea_2">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>-134</y>
              <width>742</width>
              <height>595</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QGroupBox" name="groupBox">
               <property name="title">
                <string>Information</string>
               </property>
               <layout class="QFormLayout" name="formLayout_2">
                <property name="fieldGrowthPolicy">
                 <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                </property>
                <property name="labelAlignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <item row="1" column="0">
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Render configuration file name:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLabel" name="labelRenderCfgFileName">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Film file name:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLabel" name="labelFilmFileName">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Image file name:</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLabel" name="labelImageFileName">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>Work directory:</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="QLabel" name="labelWorkDirectory">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0">
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>Start time:</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="1">
                 <widget class="QLabel" name="labelStartTime">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="6" column="0">
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>Rendering time:</string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="1">
                 <widget class="QLabel" name="labelRenderingTime">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="7" column="0">
                 <widget class="QLabel" name="label_12">
                  <property name="text">
                   <string>Samples/pixel:</string>
                  </property>
                 </widget>
                </item>
                <item row="7" column="1">
                 <widget class="QLabel" name="labelSamplesPixel">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="8" column="0">
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>Samples/sec:</string>
                  </property>
                 </widget>
                </item>
                <item row="8" column="1">
                 <widget class="QLabel" name="labelSamplesSec">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_2">
               <property name="title">
                <string>Configuration</string>
               </property>
               <layout class="QFormLayout" name="formLayout_3">
                <property name="labelAlignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Halt at samples/pixel (0 disabled):</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="lineEditHaltSPP">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Halt at time (in secs, 0 disabled):</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="lineEditHaltTime">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Film update period (in secs):</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="lineEditFilmUpdatePeriod">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Statistics print period (in secs):</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLineEdit" name="lineEditStatsPeriod">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_3">
               <property name="title">
                <string>Commands</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="2">
                 <widget class="QPushButton" name="pushButtonForceFilmMerge">
                  <property name="text">
                   <string>Force film merge</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="0" column="4">
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="0" column="1">
                 <widget class="QPushButton" name="pushButtonForceFilmDownload">
                  <property name="text">
                   <string>Force film download</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="3">
                 <widget class="QPushButton" name="pushButtonFinishJob">
                  <property name="text">
                   <string>Finish job</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_4">
               <property name="title">
                <string>Rendering</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_4">
                <item row="0" column="0">
                 <widget class="QScrollArea" name="scrollArea">
                  <property name="widgetResizable">
                   <bool>true</bool>
                  </property>
                  <widget class="QWidget" name="scrollAreaWidgetContents_4">
                   <property name="geometry">
                    <rect>
                     <x>0</x>
                     <y>0</y>
                     <width>696</width>
                     <height>76</height>
                    </rect>
                   </property>
                   <layout class="QGridLayout" name="gridLayout_5">
                    <item row="0" column="0">
                     <widget class="QLabel" name="labelRenderingImage">
                      <property name="text">
                       <string>Waiting for film download and merge</string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignCenter</set>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="queuedJobs">
        <attribute name="title">
         <string>Queued jobs</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="3">
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pushButtonAddJob">
           <property name="text">
            <string>Add job</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="4">
          <widget class="QScrollArea" name="scrollAreaQueuedJobs">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>758</width>
              <height>374</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="pushButtonRemovePendingJobs">
           <property name="text">
            <string>Remove pending jobs</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="nodes">
        <attribute name="title">
         <string>Nodes</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="0">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="3">
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pushButtonAddNode">
           <property name="text">
            <string>Add node</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="4">
          <widget class="QScrollArea" name="scrollAreaNodes">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents_3">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>758</width>
              <height>374</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="pushButtonRefreshNodesList">
           <property name="text">
            <string>Refresh nodes list</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QTextEdit" name="textEditLog">
       <property name="undoRedoEnabled">
        <bool>false</bool>
       </property>
       <property name="lineWrapMode">
        <enum>QTextEdit::NoWrap</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>lineEditHaltSPP</tabstop>
  <tabstop>lineEditHaltTime</tabstop>
  <tabstop>lineEditFilmUpdatePeriod</tabstop>
  <tabstop>lineEditStatsPeriod</tabstop>
  <tabstop>pushButtonForceFilmDownload</tabstop>
  <tabstop>pushButtonForceFilmMerge</tabstop>
  <tabstop>pushButtonFinishJob</tabstop>
  <tabstop>pushButtonAddJob</tabstop>
  <tabstop>pushButtonRemovePendingJobs</tabstop>
  <tabstop>pushButtonAddNode</tabstop>
  <tabstop>tabWidgetMain</tabstop>
  <tabstop>scrollAreaQueuedJobs</tabstop>
  <tabstop>scrollArea_2</tabstop>
  <tabstop>scrollAreaNodes</tabstop>
  <tabstop>textEditLog</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>clickedQuit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonAddJob</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clickedAddJob()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>482</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditHaltSPP</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindow</receiver>
   <slot>editedHaltSPP()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditHaltTime</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindow</receiver>
   <slot>editedHaltTime()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditFilmUpdatePeriod</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindow</receiver>
   <slot>editedFilmUpdatePeriod()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditStatsPeriod</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindow</receiver>
   <slot>editedStatsPeriod()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonForceFilmMerge</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clickedForceFilmMerge()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>438</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonForceFilmDownload</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clickedForceFilmDownload()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>438</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonFinishJob</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clickedFinishJob()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>438</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonRemovePendingJobs</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clickedRemovePendingJobs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>469</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonAddNode</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clickedAddNode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>482</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonRefreshNodesList</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clickedRefreshNodesList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>469</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>clickedQuit()</slot>
  <slot>clickedAddJob()</slot>
  <slot>editedHaltSPP()</slot>
  <slot>editedHaltTime()</slot>
  <slot>editedFilmUpdatePeriod()</slot>
  <slot>editedStatsPeriod()</slot>
  <slot>clickedForceFilmMerge()</slot>
  <slot>clickedForceFilmDownload()</slot>
  <slot>clickedFinishJob()</slot>
  <slot>clickedRemovePendingJobs()</slot>
  <slot>clickedAddNode()</slot>
  <slot>clickedRefreshNodesList()</slot>
 </slots>
</ui>
